common/migrations/0001_create_superuser.py

```diff
@@ -0,0 +1,27 @@
+"""Module for creating DB-admin."""
+from django.db import migrations
+from django.contrib.auth.hashers import make_password
+from blog.settings import ADMIN_EMAIL
```

Comment:
https://docs.djangoproject.com/en/1.10/topics/settings/#using-settings-in-python-code